### PR TITLE
Plane details, take over handicap factor from list selection. 

### LINKED ISSUE #1132: Plane Details, list selected data items not all copied
XCSoar versions having and not having the problem
-------------------------------------------------
Head rev of today. Linux desktop, sim mode.

Expected behavior
-----------------
Choose a glider type from the internal polar list should copy all data into the plane profile.

Actual behavior
---------------
The selection misses to copy the handicap factor as well as the max. cruise speed. Because the "New" button shows the form prefilled with the current active data, this might slip through the users eye and results in a wrong plane data mix.

Steps to reproduce the behavior
-------------------------------
Config -> Plane easy and always to reproduce.

Do you have any idea what may have caused this?
-----------------------------------------------
The latest change/augmentation of the stored polar data in terms of masses might have missed out something.

Do you have an idea how to solve the issue?
-------------------------------------------
Yes

### PR DESCRIPTION
**Pull request by hjr**
Brief summary of the changes
----------------------------
Copy one more item from the polar list selection.

Related issues and discussions
------------------------------
Closes #1132

### CODE REVIEW

**Comment by hjr** [comment]
Ok, dont know why this Austria polar slipped in here. Stupid git.

File: src/Dialogs/Plane/PlaneDetailsDialog.cpp [hunk 1]
```diff
@@ -185,6 +185,7 @@ PlaneEditWidget::PolarButtonClicked() noexcept
     LoadValue(TYPE, plane.polar_name.c_str());
 
   /* reload attributes that may have been modified */
+  LoadValue(HANDICAP, static_cast<int>(plane.handicap));
```
Comment (MaxKellermann) [inline src/Dialogs/Plane/PlaneDetailsDialog.cpp:188]: Let's not do this cast, this is fragile code clutter. Instead, `RowFormWidget()` should have an (inline) overload for `unsigned` which does this cast.

Comment (hjr) [reply src/Dialogs/Plane/PlaneDetailsDialog.cpp:188]: agreed

Comment (MaxKellermann) [reply src/Dialogs/Plane/PlaneDetailsDialog.cpp:188]: I've added the overload, you can now amend this commit to remove the cast.

Comment (hjr) [reply src/Dialogs/Plane/PlaneDetailsDialog.cpp]: Thx, did so.

**Pull request merged**